### DISCUSSION TITLE
Loco nodes can not be recognized

### DISCUSSION THREAD

**Discussion by Chilingyue** [bitcraze/discussions#804] [Q&A] [answered]
my nodes was connected, but in Anchor position configuration there was no anchor to see ,there was nothing ,I can not get location from anchors. All the  8 anchors was green already.
![image](https://github.com/bitcraze/discussions/assets/130750514/206640e4-0cff-465c-846f-b3ccac920392)

**Comment by krichardsson** [comment]
Hi!

The anchor ids are unfortunately not added automatically, you have to add them manually by clicking the "Add anchor" button and entering the id.

**Comment by Chilingyue** [comment]
We  wrote it by "add anchor", but after adding nodes and then pressing Get from anchors, the added anchors disappeared and like the picture above

**Comment by krichardsson** [comment, marked as answer]
Sorry, I was not clear in my previous answer.

The reason you don't get any anchors in the dialog is because there is not yet any position stored in the anchors. What you need to do is:

1. Click `Add anchor`
2. Enter `0` for the first anchor and click OK
3. Now you should have a new row in the dialog for anchor 0. Enter the coordinates for the anchor position.
4. Repeat steps 1 - 3 for all anchors
5. Click `Write to anchors` to write the data to the anchors
6. After a while the boxes should turn green. This happens when the position has been stored in the anchor and a UWB packet (with the position) has been received by the Crazyflie, which in turn has sent the position to the Client. If one or more boxes do not turn green, you might have to move the Crazyflie closer to those anchors and click `Write to anchors` again

**Reply by CalvinKoung** [reply]
so in the first time, I need to mannul add the anchor position ? but how can I know the anchor definitly position ? tks

**Reply by ataffanel** [reply]
You need to physically measure where you place the anchors in your space and enter the position for each of them.

**Comment by Chilingyue** [comment]
Thank you so much, so "get from anchors" is not useful, which is different from the official tutorial

**Reply by krichardsson** [reply]
"get from anchors" works when there is already a position stored in the anchors, but not  when the anchors are new.
We will take a look at the tutorial and update it.
